app/src/main/java/com/example/concordia_campus_guide/Models/BusStop.java

```diff
@@ -0,0 +1,22 @@
+package com.example.concordia_campus_guide.Models;
+
+import com.example.concordia_campus_guide.ClassConstants;
+
+public class BusStop extends Place {
```

Comment:
Why did we create a specific class for the BusStop ? can't we use it the walkingpoint class ? by for example, having a pointType BusStop . What do you think?

Reply:
Again we need a Place

Reply:
If it is not a walking place, we should remove it from the walking_points.json

Reply:
Unless it'll be used for seemless integration b/w indoor+shuttle integration

Reply:
I think it can be removed since we're using the coordinates in the ClassConstants